### DISCUSSION TITLE
How to check what is happening?

### DISCUSSION THREAD

**Discussion by DarkCorner-IT** [firewalld/firewalld#1332] [Q&A]
FirewallD is active on my server. I didn't activate it or even configure it.
I add that I don't know it and have never worked with it.

How do I know if it's working properly?
How can I see who attacked what, where and how the firewall intervened?

**Comment by erig0** [comment]
> How do I know if it's working properly?

You can check the running state with `firewall-cmd --state`. It will say `running` if firewalld is running normally or `failed` if the firewall is in failed state (e.g. config error).

> How can I see who attacked what, where and how the firewall intervened?

Firewalld denies most traffic by default. You can log dropped connection attempts/packets with `--set-log-denied=all`. Results will be in the kernel log buffer; use `dmesg` to see them.
